src/scss/components/pages.scss

```diff
@@ -79,16 +79,38 @@
 
 // Page sidebar
 .page-sidebar-header {
-  border-bottom: 1px solid $gray-600;
-  padding-bottom: 8px;
+  border-bottom: 1px solid $sidebar-bottom-border-color;
```

Comment:
Rather than introduce a new variable just for this border, I'm just wondering if one of the pre-existing ones would work ... $gray-500 seems a pretty close match to me.

Reply:
Done!